builder/builder-next/worker/label/label.go

```diff
@@ -0,0 +1,9 @@
+package label
+
+// Pre-defined label keys similar to BuildKit ones
+// https://github.com/moby/buildkit/blob/master/worker/label/label.go
```

Comment:
Can you change the link to a permalink? https://github.com/moby/buildkit/blob/v0.11.6/worker/label/label.go#L3-L16

Should we consider adding a const in BuildKit as well? (and a TODO here to replace it).


Should this option be limited to this use specific, or are there other cases where custom host entries must be defined on the worker? i.e., could could the label be a map, or (e.g.) a prefix; `org.mobyproject.buildkit.worker.hosts.host-gateway`, `org.mobyproject.buildkit.worker.hosts.hello-world`

(not sure if a good idea, or if that would be a YANI but happy to hear thoughts)

Reply:
> Can you change the link to a permalink? https://github.com/moby/buildkit/blob/v0.11.6/worker/label/label.go#L3-L16

:+1:

> Should we consider adding a const in BuildKit as well? (and a TODO here to replace it).

It's specific to moby workers so don't think we should.

> Should this option be limited to this use specific,

Current case is already edgy by abusing worker labels so no don't think we should.